DataFormats/PatCandidates/src/classes_def_objects.xml

```diff
@@ -514,6 +514,20 @@
   <class name="edm::Wrapper< std::vector<edm::Ptr<pat::Jet> > >" />
   <class name="edm::Wrapper< std::vector< std::vector<edm::Ptr<pat::Jet> > > >" />
 
+  <class name="pat::PATTauDiscriminatorBase">
+    <field name="transientVector_" transient="true"/>
+  </class>  
+  <class name="pat::PATTauDiscriminator"  ClassVersion="3">
+   <version ClassVersion="3" checksum="2916098408"/>  
+  </class>  
```

Comment:
it may be true, that the derived class, the new data members, are indeed seen as regular class. Also, we have it in many places, pat:: is at least one example.
So, please go ahead with the versions as you have already.